src/modules/job-info/job_state.c

```diff
@@ -443,7 +443,20 @@ static int eventlog_lookup_parse (struct info_ctx *ctx,
                                 __FUNCTION__, (uintmax_t)job->id);
                 goto nonfatal_error;
             }
-            break;
+        }
+        else if (!strcmp (name, "priority")) {
+            if (!context) {
+                flux_log_error (ctx->h, "%s: no priority context for %ju",
+                                __FUNCTION__, (uintmax_t)job->id);
+                goto nonfatal_error;
+            }
+
+            if (json_unpack (context, "{ s:i }",
+                             "priority", &job->priority) < 0) {
+                flux_log_error (ctx->h, "%s: priority context for %ju invalid",
+                                __FUNCTION__, (uintmax_t)job->id);
+                goto nonfatal_error;
+            }
```

Comment:
`json_unpack()` doesn't set errno, so don't use `flux_log_error()`.

Reply:
Not to get too off track but the identical code right above it has the same problem. Fix here?

Reply:
doh!  I could have swore i fixed these.  perhaps i rebased wrong.